### DISCUSSION TITLE
Amd Gpus?

### DISCUSSION THREAD

**Discussion by Qel0droma** [Purfview/whisper-standalone-win#14] [Q&A] [answered]
Hello, thanks for all the hard work! This model works wonderfully on NVidia GPUs, although it seems to be less accurate with Japanese audio, most likely due to faster whisper. Anyways, my question is: will this also work with AMD GPUs, such as the 6700XT? Thanks!

**Comment by Purfview** [comment, marked as answer]
No, AMD GPUs doesn't have CUDA.
